.clj-kondo/config.edn

```diff
@@ -478,7 +478,6 @@
   metabase.models.json-migration/def-json-migration                                    clj-kondo.lint-as/def-catch-all
   metabase.models.setting.multi-setting/define-multi-setting                           clojure.core/def
   metabase.models.setting/defsetting                                                   clj-kondo.lint-as/def-catch-all
-  metabase.public-settings.premium-features/defenterprise                              clj-kondo.lint-as/def-catch-all
```

Comment:
I couldn't make the LSP linter for this stop complaining so I gave up and wrote a custom hook for it. The custom hook works better anyway since it properly analyzes the body of the `defenterprise` form which it did not do before.